### PR TITLE
Add mirandachrist as an adoree

### PR DESCRIPTION
**Pull request by mirandachrist**
Also removes the laugh emoji, which can be interpreted as mean
/assign @fejta

### CODE REVIEW

File: prow/plugins/heart/heart.go [hunk 1]
```diff
@@ -38,7 +38,6 @@ const (
 
 var reactions = []string{
 	github.ReactionThumbsUp,
-	github.ReactionLaugh,
```
Comment (fejta) [inline prow/plugins/heart/heart.go]: Finally!

**Comment by mirandachrist** [comment]
Does this work yet?

**Comment by mirandachrist** [comment]
Apparently so!

**Pull request merged**